DEPS

```diff
@@ -22,4 +22,3 @@ DOTNET_SDK_IMAGE=mcr.microsoft.com/dotnet/sdk:${DOTNET_SDK_VERSION}
 DOTNET_RUNTIME_IMAGE=mcr.microsoft.com/dotnet/runtime:${DOTNET_SDK_VERSION}
 PYTHON_IMAGE_VERSION=3.8-alpine3.15
 PYTHON_IMAGE=python:${PYTHON_IMAGE_VERSION}
-MO_URL=https://raw.githubusercontent.com/tests-always-included/mo/master/mo
```

Comment:
I think it could be valuable to not download `mo` and instead `cp` it, but we'd still need all the `install-mo` things to get them into the Docker image and it would be a little bit MORE complex, because we need to get them into the right place in the filesystem. It might be possible to symlink all the uses of `mo` and then when the Docker build bundle is made, it will resolve to the real thing. If you want, you can make a card for that or just do it